x-pack/plugins/lists/public/exceptions/hooks/persist_exception_item.tsx

```diff
@@ -43,11 +45,20 @@ export const usePersistExceptionItem = ({
       if (exceptionListItem != null) {
         try {
           setIsLoading(true);
-          await persistExceptionItem({
-            http,
-            listItem: exceptionListItem,
-            signal: abortCtrl.signal,
-          });
+          if (isUpdateExceptionItem(exceptionListItem)) {
```

Comment:
NOTE: Separated out calls since they now each validate the expected request and response schemas, which were different for PUT/POST.